charts/odigos/values.yaml

```
@@ -1,5 +1,5 @@
 image:
-  tag: v1.0.47
+  tag: v1.0.49
```

since we already released v1.0.49:

```suggestion
  tag: v1.0.50
```